README.md

```diff
@@ -71,4 +94,39 @@ def main:
     print (2 + 2)
 ```
 
-To compile and run the project, simply type `luna` in the main project directory.
+To compile and run the project, simply type `luna` in the main project 
+directory.
+
+## Contributing to Luna
+If you are interested in contributing to the development of Luna, please read
+the 
+[`CONTRIBUTING.md`](https://github.com/luna/luna/blob/master/CONTRIBUTING.md)
+file. 
+
+## License
+This repository is licensed under the
+[Apache 2.0](https://opensource.org/licenses/apache-2.0), as specified in the
+[LICENSE](https://github.com/luna/luna/blob/master/LICENSE) file. All
+contributions to this repository are accepted under these terms.
+
+Please be aware that, as the commercial backing for Luna, 
+**New Byte Order Sp. z o. o.** reserves the right under the Apache 2.0 license 
+to use contributions made to this repository as part of commercially available
+Luna products. 
+
+If these terms are unacceptable to you, please do not contribute to the 
+repository.
+
+### Contributions
```

Comment:
While this will most probably remain true, I'd like to not state this here. What if we decided to MIT license it at some point? The CLA leaves just enough wiggle room for such changes, so I'd remove this section.